std_msgs/Header header
PredictedGoal[] goals